app/src/main/java/org/schabi/newpipe/fragments/list/comments/CommentsFragment.java

```diff
@@ -84,16 +84,14 @@ public void showLoading() {
     public void handleResult(@NonNull final CommentsInfo result) {
         super.handleResult(result);
 
-        AnimationUtils.slideUp(requireView(), 120, 150, 0.06f);
+        ViewUtils.slideUp(requireView(), 120, 150, 0.06f);
 
         if (!result.getErrors().isEmpty()) {
             showSnackBarError(result.getErrors(), UserAction.REQUESTED_COMMENTS,
                     NewPipe.getNameOfService(result.getServiceId()), result.getUrl(), 0);
         }
 
-        if (disposables != null) {
-            disposables.clear();
-        }
+        disposables.clear();
```

Comment:
removing the null check won't cause problems?

Reply:
The variable is final, so there won't be any problems.